README.md

```diff
@@ -1,2 +1,115 @@
+3. Run "IIS"
+4. Create a new website e.g. 'Acme'. In the **Add website** dialog:
+	* Make sure the physical path is pointing to the folder where the unzipped content of umbraco are placed e.g. "C:\Umbraco_8_1_1" in our case.
+	* Provide a host name e.g. 'Acme'
+	* Keep all other settings as they are for basic setup.
+	* Click OK
+5. **Note**: The web site will not run when you will browse "http://Acme" or "http://Acme/Umbraco". You will have to register this host name "Acme" with local IP i.e. 127.0.0.1
+in the *(C:\Windows\System32\drivers\etc\host)* file. (Make sure you edit it in Admin mode)
+6. Once registered to host file, browse to the Acme. You will be presented with same installation screens as described above under **"When IIS is not installed locally"** section on step 10 onwards.
+7. Install Umbraco same as step 10 onward in section above or as per your needs.   
+
+---
+## Visual Studio Code Setup
+* Clone the the code from [github](https://github.com/KamranIE/Acme.git)
+* Create a feature branch from master. e.g. feature/NewAthletePage
+* Switch to the new branch
+* You will find following solution with projects 
+	1. Acme.sln
+	2. Acme.UI
+	3. Acme.Umbraco.Models
+	
+The details of the 3 is as follows:
+### Acme.sln
+It is the solution that contains the two projects "Acme.UI" and "Acme.Umbraco.Models". Always open this solution file in visual studio whenever you want to modify any of 
+the two projects.
+
+### Acme.UI Project
+It is an ASP.Net MVC project and contains all the custome controllers, views and out custom handlers to the umbraco request pipeline. 
+Please setup following after opening this project:
+1. In the Project properties "Build Events" copy following lines to "Pre-build event command line:" 
+`tasklist /FI "IMAGENAME eq VBCSCompiler.exe" 2>NUL | find /I /N "VBCSCompiler.exe">NUL
+if "%ERRORLEVEL%"=="0" (taskkill /IM VBCSCompiler.exe /F) else (verify >NUL)`
+_**Note: Above lines are necessary to kill the VBCSCompiler before you start building. Otherwise you will encounter build/publish issues as this process locks the files every time you compile and for some reason doesn't release the lock most of the times**_
+2. You need to setup a publish profile as follows to help you publish to the right folder and debug. *(It should already be there - just modify it as per your settings if it is)*
+	1. Name e.g. 'LocalDev'
+	2. Publish Method: File System
+	3. Target location: <Umbraco web site e.g. C:\Projects\Acme\UmbracoCms.8.1.1>
+	4. Configuration: Debug
+### Acme.Umbraco.Models Project
+* It contains all the files from Umbraco models folder (e.g. \Acme\UmbracoCms.8.1.1\App_Data\Models in our case). 
+* __Important:__ The files from models folder are not automatically copied and included to the project. You will have to include any new changes to the project manually.
+ 
+---
+## Git setup
```

Comment:
I would remove this, its self documenting in the git file that is also checked in